### DISCUSSION TITLE
WinUI 3 Updated to 1.1! Input validation grayed out?

### DISCUSSION THREAD

**Discussion by navarone77** [microsoft/WinUI-Gallery#991] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/97413201/172260800-69c2890f-9b78-4727-bc3c-c7c943cbfc2c.png)

I'm missing Input Validation , does anyone else have this ?

  I wasn't sure if this is a bug or was disabled on purpose.  If I'm not mistaken I don't recall this being grayed out in previous version,  Am I missing something?

Hope to get this confusion cleared up .
Thanks for reading this post.

**Comment by gregwoo-microsoft** [comment, marked as answer]
Thanks for your message @navarone77. The Input Validation sample was disabled on purpose as the page doesn't display anything at the moment. We plan on implementing this page for a future iteration of the app.

**Reply by navarone77** [reply]
Oh okay makes sense, thank you for the info!
